book/src/supplychain-security/signing-windows.md

```diff
@@ -57,15 +55,15 @@ Want support for another vendor? [Drop us a line](mailto:hello@axo.dev) or [file
     For reference, the [SSL.com documentation for their official GitHub Action](https://www.ssl.com/how-to/cloud-code-signing-integration-with-github-actions/) uses the names: ES_USERNAME, ES_PASSWORD, ES_TOTP_SECRET, and CREDENTIAL_ID for these values. The “ES” stands for “esign”. We renamed these variables to make them more specific and clear.
 
 
-#### Part 2: Enable SSL.com signing with cargo-dist
+### Part 2: Enable SSL.com signing with cargo-dist
 
 1. **Configure cargo-dist to codesign**
 
     Add the following to your `Cargo.toml` or `dist.toml`:
 
     ```toml
     [workspace.metadata.dist]
-    ssldotcom-windows-sign = "prod" # or "test" if you are using a sandbox account
+    ssldotcom-windows-sign = "prod"  or "test" if you are using a sandbox account
```

Comment:
@Gankra is this on purpose?

Reply:
Sure wasn't

Reply:
oh lmao i did `s/# /` to lower the level of each heading by one